helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/HelpDeskBotStories.java

```diff
@@ -108,4 +124,43 @@ public InjectableStepsFactory stepsFactory() {
         "**/*.story", "**/excluded*.story");
   }
 
+  private Room getQueueRoom() {
```

Comment:
Please include javadoc to explain the behavior of this method.

Reply:
Done.